build/commands/lib/l10nUtil.js

```diff
@@ -193,7 +194,8 @@ const l10nUtil = {
         braveSpecificGeneratedResourcesPath,
         braveResourcesComponentsStringsPath,
         braveExtensionMessagesPath,
-        braveAndroidBraveStringsPath
+        braveAndroidBraveStringsPath,
```

Comment:
@goodov seems error prone to have to manually add these, we should maybe just search for patterns like `*_strings.grd`?